src/components/settings-menu/DevNotificationsSection.tsx

```diff
@@ -1,55 +1,23 @@
-import messaging from '@react-native-firebase/messaging';
 import React, { useEffect, useMemo, useState } from 'react';
 import { ScrollView } from 'react-native-gesture-handler';
 import { MiniButton } from '../buttons';
 import { ListFooter } from '../list';
+import {
+  subscribeToTransactionTopics,
+  unsubscribeFromTransactionTopics,
+} from '@/notifications/subscriptions';
 import { Box, Columns, Text } from '@/design-system';
 import { useAccountSettings, useWallets } from '@/hooks';
 import { useTheme } from '@/theme';
 import { formatAddressForDisplay } from '@/utils/abbreviations';
 
-const topics = [
-  'sent',
-  'received',
-  'purchased',
-  'sold',
-  'minted',
-  'swapped',
-  'approvals',
-  'other',
-];
-
-const firebaseSubscribeTopics = async (
-  type: string,
-  chainId: number,
-  address: string
-) => {
-  topics.forEach(topic => {
-    messaging().subscribeToTopic(
-      `${type}_${chainId}_${address.toLowerCase()}_${topic}`
-    );
-  });
-};
-
-const firebaseUnsubscribeTopics = async (
-  type: string,
-  chainId: number,
-  address: string
-) => {
-  topics.forEach(topic => {
-    messaging().unsubscribeFromTopic(
-      `${type}_${chainId}_${address.toLowerCase()}_${topic}`
-    );
-  });
-};
-
 const DevNotificationsSection = () => {
   const { colors } = useTheme();
   const { wallets } = useWallets();
   const { chainId } = useAccountSettings();
   const [loading, setLoading] = useState<boolean>(true);
   const walletIDs = Object.keys(wallets!);
-  let allWallets: any[] = useMemo(() => [], []);
+  const allWallets: any[] = useMemo(() => [], []);
```

Comment:
I expect you don't want to use it, because this might randomly refresh clearing the list according to docs of react. 

You might want ot use instead:

- `useMemoOne(() => [], [])`
- `useRef([]).current`
- `useState([])[0]`